rust/operator-binary/src/superset_controller.rs

```diff
@@ -713,7 +714,7 @@ fn build_server_rolegroup_statefulset(
             .with_label("restarter.stackable.tech/enabled", "true")
             .build(),
         spec: Some(StatefulSetSpec {
-            pod_management_policy: Some("Parallel".to_string()),
+            pod_management_policy: Some("OrderedReady".to_string()),
```

Comment:
Whats the reason for this?

Reply:
Could we add a comment for why it is set? Exactly the part with the DB init / upgrade?

Reply:
Yes, good point, comment added.